CMakeLists.txt

```diff
@@ -68,7 +119,7 @@ set (MOD_VCF_SOURCES
         inc/vcf/validator_detail_v42.hpp
         inc/vcf/validator_detail_v43.hpp
         inc/vcf/validator.hpp
-        
+
```

Comment:
Remove blank line.

Reply:
The line was there earlier also. I have just removed the trailing white space from it.
This line was just to break those all header files into two blocks just for readability.
should we remove that line?